ui/src/pages/case.triggers.js

```
@@ -0,0 +1,93 @@
+document.addEventListener("DOMContentLoaded", () => {
+    loadTableData();
+    const table = document.getElementById("trigger_table");
+    const responseModal = document.getElementById("responseModal");
+    const closeButton = responseModal.querySelector('[data-dismiss="hide-frame"]');
+    
+    // Initialize the modal instance
+    const modal = new bootstrap.Modal(responseModal);
+ 
+    // Show modal on "View Response" button click
+    table.addEventListener("click", (event) => {
+        if (event.target && event.target.classList.contains("view-response")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)